meshroom/ui/qml/Viewer3D/TransformGizmo.qml

```diff
@@ -541,6 +550,131 @@ Entity {
                 }
             }
 
+            // MOVE SURFACE ENTITY (surfaces on axis direction)
+            Entity {
+                id: surface1MoveEntity
```

Comment:
Can we rename both entities ids to something more explicit ? Also, can you add a line or two explaining what is a "surface on axis direction". It may be clear in your head, but not for the next reader.